src/components/Player/Player.scss

```diff
@@ -0,0 +1,9 @@
+.player {
+  background: #5b50ff;
+  color: white;
+  padding: 4rem;
+  position: fixed;
```

Comment:
component should know nothing about its positioning. Please move positioning rules to the main component